src/Article.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import './Article.css';
+import featuredimage from './image-3-medium.jpg';
+
+class Article extends Component {
+  render() {
+  	const toggledClass = this.props.expanded ? 'expanded' : 'collapsed';
+    var image = require(`${this.props.article.featured_image}`);
+    return (
+      <div className="c-mobile-article">
+      	<div className={`c-mobile-article__content ${toggledClass}`}>
+          <h1>{this.props.article.headline}</h1>
+          <div className="c-mobile-article__info">
+            <div className="c-mobile-article__info--left">
+              <div className="c-mobile-article__author"><b>Written by {this.props.article.author}</b></div>
+            </div>
+            <div className="c-mobile-article__info--right">
+              <div className="c-mobile-article__date">
+                {new Intl.DateTimeFormat('en-US', {
```

Comment:
Can this be moved to a helper? It's just a little messy when it's in with the JSX.